MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -3956,35 +3956,32 @@ private void assignToCrewSlot(Person p, int slot, String gunType, String driveTy
     }
 
     public void resetEngineer() {
-        if(!isSelfCrewed()) {
+        if (!isSelfCrewed()) {
             return;
         }
         int minutesLeft = TECH_WORK_DAY;
         int overtimeLeft = TECH_WORK_DAY / 2;
-        int edgeLeft = 0;
         boolean breakpartreroll = true;
         boolean failrefitreroll = true;
-        if(null != engineer) {
+        if (null != engineer) {
             minutesLeft = engineer.getMinutesLeft();
             overtimeLeft = engineer.getOvertimeLeft();
-            edgeLeft = engineer.getEdge();
         } else {
             //then get the number based on the least amount available to crew members
             //in the case of Edge, everyone must have the same triggers set for Edge to work
-            for(Person p : getActiveCrew()) {
-                if(p.getMinutesLeft() < minutesLeft) {
+            for (Person p : getActiveCrew()) {
+                if (p.getMinutesLeft() < minutesLeft) {
                     minutesLeft = p.getMinutesLeft();
                 }
-                if(p.getOvertimeLeft() < overtimeLeft) {
+
+                if (p.getOvertimeLeft() < overtimeLeft) {
                     overtimeLeft = p.getOvertimeLeft();
                 }
-                if(p.getEdge() < edgeLeft) {
-                    edgeLeft = p.getEdge();
```

Comment:
What was this for?

Reply:
Completely unused, so I removed the logic.